velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -181,3 +190,97 @@ TEST_F(GreatestLeastTest, clearNulls) {
     }
   }
 }
+
+TEST_F(GreatestLeastTest, shortDecimal) {
+  const auto shortType = DECIMAL(10, 4);
+  const std::vector<std::optional<int64_t>> firstColumnUnscaled = {
```

Comment:
This variable is used only once and is not needed. Let's remove it. This will also make initialization of inputs and expected results consistent.